.changeset/swift-rabbits-appear.md

```diff
@@ -0,0 +1,34 @@
+---
+"@latticexyz/explorer": patch
+---
+
+### Initial Release of Explorer
+
+Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
```

Comment:
We've generally avoided the use of headlines in changesets because the changelog makes use of headlines and the changeset including them mucks up formatting a bit.

Can we frame this in the context of a changeset?

> Initial release of the `@latticexyz/explorer` package. Explorer is a standalone tool designed to explore and manage worlds. This initial release includes support for only local worlds, with support for any world in the near future.

Reply:
Yeah, like your description, changed and removed the title.